app/src/test/java/com/x8bit/bitwarden/data/autofill/processor/AutofillProcessorTest.kt

```diff
@@ -179,6 +262,8 @@ class AutofillProcessorTest {
                 request = fillRequest,
             )
 
+            testDispatcher.scheduler.runCurrent()
```

Comment:
So this is the only other test that needs this because its the only one that runs the coroutine?

Reply:
Yep, Only the call to `processFillRequest` with a fillable request runs a coroutine.